server/src/main/java/org/elasticsearch/node/Node.java

```diff
@@ -1951,6 +1951,13 @@ protected ScriptService newScriptService(
         return new ScriptService(settings, engines, contexts, timeProvider);
     }
 
+    /**
+     * Creates a new the ReadinessService. This method can be overwritten by tests to inject mock implementations.
```

Comment:
```suggestion
     * Creates a new ReadinessService. This method can be overwritten by tests to inject mock implementations.
```

Reply:
Copy paste :) 
Will fix it in the other javadocs too